api/src/main/resources/db/data/afterMigrate.sql

```diff
@@ -0,0 +1,3 @@
+USE nextdoor;
+-- USER
+INSERT IGNORE INTO user VALUES ('3b13da2d-0ff8-4388-88fb-5182b4e4ec42','2019-12-18 13:17:17','test@test.test','password'); -- Bcrypt encrypted: $2y$12$UrKngInmATHE5X4PPL/oEuqzWi.nmVsQLhd94lUoBGDTKaqzwJv8W
```

Comment:
the password needs to be encoded before saved to DB

Reply:
Updated with encrypted value.